platform/kindle/extensions/koreader/README.txt

```diff
@@ -13,13 +13,13 @@ K4 Yes
 Touch Yes
 PW Yes
 PW2 Yes
-KT Yes
 Voyage Yes
 PW3 Yes
 KOA Yes
 KT2 Yes
 KOA2 Yes
 PW4 Yes
+KT3 Yes
```

Comment:
Not a Kindle owner but the grouping seems a touch random to me. If it's by date I'm still not sure it plain alphabetical wouldn't be easier to use.

Reply:
Agree. I think the list order in alphabetical will be easier to read.

Reply:
Never mind, I'm contradicting @NiLuJe